lib/Cake/Utility/Validation.php

```diff
@@ -561,6 +561,28 @@ public static function maxLength($check, $max) {
 	}
 
 /**
+ * Checks whether the length of a string (in bytes) is greater or equal to a minimal length.
+ *
+ * @param string $check The string to test
+ * @param int $min The minimal string length
+ * @return bool Success
+ */
+	public static function minLengthBytes($check, $min) {
+		return strlen($check) >= $min;
```

Comment:
strlen returns number of characters and not bytes. If you want bytes a Unicode string is using, then you have to do this.
```
    return mb_strlen($check, '8bit') >= $min;
```